core/internal/cltest/simulated_clock.go

```diff
@@ -0,0 +1,28 @@
+package cltest
```

Comment:
This might be better places in testutils not cltest

Reply:
Thanks, done.